cmd/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	TXIndex              bool
 	Regtest              bool
 	PostgresURI          string
+	PosValidatorSeed     string
```

Comment:
Seconded!

Reply:
Check my comment re `viper.GetSting()`

Reply:
Leaving it as PosValidatorSeed seems fine since we're not sure 100% yet whether or not BIP39 seed phrase -> BLS priv key derivation works. So the seed may need to be something else. It's fine to leave it the name as-is until that's disambiguated